src/i18n/de/openSystem.json

```diff
@@ -0,0 +1,2580 @@
+      }
+    },
+    "journalEntryType": {
+      "cultDescription": {
+        "name": "Kult",
+        "title": "<h3>Kult</h3><br/>"
+      },
+      "runeDescription": {
+        "name": "Rune",
+        "title": "<h3>Rune</h3><br/>"
+      },
+      "runeMagicDescription": {
+        "name": "Runenmagie",
+        "title": "<h3>Rune Magic</h3><br/>"
+      },
+      "skillDescription": {
+        "name": "Fertigkeit",
+        "title": "<h3>Fertigkeit</h3><br/>"
+      },
+      "spiritMagicDescription": {
+        "name": "Geistermagie",
+        "title": "<h3>Spirit Magic</h3><br/>"
+      }
+    },
+    "language": {
+      "esrolian": {
+        "name": "Esrolisch"
+      },
+      "heortling": {
+        "name": "Heortlingisch"
+      },
+      "newPelorian": {
+        "name": "Neupelorisch"
+      },
+      "praxian": {
+        "name": "Praxisch"
+      },
+      "pureHorseTongue": {
+        "name": "Reine Pferdesprache"
+      },
+      "stormspeech": {
+        "name": "Sturmsprache"
+      },
+      "tarshite": {
+        "name": "Tarschitisch"
+      },
+      "tradetalk": {
+        "name": "Handelssprache"
+      }
+    },
+    "rune": {
+      "air": "Luft",
+      "beast": "Tier",
+      "chaos": "Chaos",
+      "combine": "Kombinieren",
+      "command": "Befehligen,",
+      "darkness": "Dunkelheit",
+      "dragonewt": "Drachenmolch",
+      "death": "Tod",
+      "disorder": "Unordnung",
+      "dispel": "Auflösen",
+      "earth": "Erde",
+      "fate": "Schicksal",
+      "fertility": "Fruchtbarkeit",
+      "fireSky": "Feuer/Himmel",
+      "harmony": "Harmonie",
+      "illusion": "Illusion",
+      "infinity": "Unendlichkeit",
+      "law": "Recht",
+      "light": "Licht",
+      "luck": "Glück",
+      "magic": "Magie",
+      "man": "Mensch",
+      "mastery": "Meisterschaft",
+      "moon": "Mond",
+      "movement": "Bewegung",
+      "plant": "Pflanze",
+      "separate": "Trennen",
+      "spirit": "Geist",
+      "stasis": "Stasis",
+      "summon": "Beschwören",
+      "tap": "Anzapfen",
+      "truth": "Wahrheit",
+      "water": "Wasser"
+    },
+    "runeDescription": {
+      "airElement": {
+        "name": "Luft (Element)",
```

Comment:
As you found out, the rune names unfortunately are not translatable currently. So to make the system work, they need to be the english names for now.